Scripts/TypewriterUI.cs

```diff
@@ -0,0 +1,89 @@
+// Script for having a typewriter effect for UI

```

Comment:
You can learn alot from this class, well organised and the variables are self explanatory name wise